shared/features/contributors/activity-graph/activity-graph.hooks.tsx

```
@@ -7,8 +7,27 @@ import {
 
-const useActivityGraphRange = ({ start, end }: { start?: Date; end?: Date }) => {
+const useActivityGraphRange = ({ start, end, fromDate, toDate }: { start?: Date; end?: Date; fromDate?: string; toDate?: string }) => {
   const dateKernel = bootstrap.getDateKernelPort();
+
+  // Handle fromDate/toDate if provided (new API parameters)
+  if (fromDate || toDate) {
+    const from = fromDate ? new Date(fromDate) : dateKernel.subYears(new Date(), ActivityGraphConfig.number_of_years);
+    const to = toDate ? new Date(toDate) : new Date();
+    
+    // Validate dates
+    if (isNaN(from.getTime()) || isNaN(to.getTime())) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)